app/models/claim_stats.rb

```diff
@@ -0,0 +1,11 @@
+class ClaimStats < ApplicationRecord
+  self.primary_key = "claim_id"
+
+  def readonly?
+    true
+  end
+
+  def self.refresh
+    Scenic.database.refresh_materialized_view(table_name, concurrently: false, cascade: false)
```

Comment:
The materialised view will need refreshing. However, I will do this in the upload portion (covered in the next ticket) and refresh the view as part of the background job.